lib/models/component-set-fields.ts

```diff
@@ -0,0 +1,61 @@
+/**
+ * Combines information about components, includes label, iconUrl
+ * and component related fields
+ * label and iconUrl are extracted from component-definition.json file, while fields are loaded from html templates
+ * Example:
+ * {
+ *   body: {
```

Comment:
Example could use some updating, such as the label and iconUrl.

Reply:
I updated those fields now.